### PR TITLE
More PolyBench unrolling

### PR DESCRIPTION
**Pull request by sampsyo**
I tried unrolling MVT, and it works! I am opening this PR to get a sanity check that I have done everything I was supposed to do to unroll an existing benchmark.

MVT also has an unrelated thing going on that I noticed but did not address in this PR: it uses `+=` to accumulate matrix-vector products into a couple of output vectors (`x1` and `x2`), and our data file currently provides nonzero initial data for these vectors. This means that the results are the actual products with those initial values added on. We should just start at zero to compute the "correct" `x1` and `x2`. But I left everything how it was in this PR so that we matched the test results exactly.

### CODE REVIEW

File: benchmarks/data/linear-algebra-mvt.data [hunk 1]
```diff
@@ -18,12 +18,29 @@
              [17,49,9,5,31,3,37,37]],
     "bitwidth": 32
   },
-  "x20": {
-    "data": [3,31,27,39,43,6,9,1],
+  "x1_int0":{
+    "data": [20,49,17,40,11,30,29,10],
```
Comment (rachitnigam) [inline benchmarks/data/linear-algebra-mvt.data:22]: Why did the input data change?

**Comment by rachitnigam** [comment]
LGTM everything except the one comment. Probably doesn't matter because the data in output only.

**Comment by rachitnigam** [comment]
Feel free to merge.

File: benchmarks/data/linear-algebra-mvt.data (hunk 1 above)
Comment (sampsyo) [reply benchmarks/data/linear-algebra-mvt.data:22]: It didn't—I just flipped the order of x1 and x2 (so now they appear in lexicographical order).

**Pull request merged**